spec/attrs_spec.rb

```diff
@@ -27,6 +31,16 @@ def initialize(name)
           .to(raise_error(ReturnContractError))
       end
 
+      it "getting valid type for second val" do
+        expect(Person.new("bob").name_r_2)
+          .to(eq("bob"))
+      end
+
+      it "getting invalid type for second val" do
+        expect { Person.new(1.3).name_r_2 }
+        .to(raise_error(ReturnContractError))
```

Comment:
Rubocop expects this to be indented two spaces: https://travis-ci.org/egonSchiele/contracts.ruby/jobs/234892156